### ISSUE TITLE
Using stable_baselines3 leads to the following error

### ISSUE THREAD

**Issue by wilhem** [aidudezzz/deepbots-tutorials#43]
I installed today `stable_baselines3` using pip.
The installation went good. Unexpected the pip installation process removed `gymnasium 0.29.1`and installed `gymnasium 1.0.0a2` and `shimmy-2.0.0`

Now the problem is that trying to import stable_baselines3 in the code gives me the following error:
```

Traceback (most recent call last):
  File "/home/ubuntu/Downloads/my_project/controllers/cartpole_v2/cartpole_v2.py", line 5, in <module>
    import stable_baselines3
  File "/home/ubuntu/Downloads/deepbots-env/lib/python3.10/site-packages/stable_baselines3/__init__.py", line 3, in <module>
    from stable_baselines3.a2c import A2C
  File "/home/ubuntu/Downloads/deepbots-env/lib/python3.10/site-packages/stable_baselines3/a2c/__init__.py", line 1, in <module>
    from stable_baselines3.a2c.a2c import A2C
  File "/home/ubuntu/Downloads/deepbots-env/lib/python3.10/site-packages/stable_baselines3/a2c/a2c.py", line 7, in <module>
    from stable_baselines3.common.buffers import RolloutBuffer
  File "/home/ubuntu/Downloads/deepbots-env/lib/python3.10/site-packages/stable_baselines3/common/buffers.py", line 17, in <module>
    from stable_baselines3.common.vec_env import VecNormalize
  File "/home/ubuntu/Downloads/deepbots-env/lib/python3.10/site-packages/stable_baselines3/common/vec_env/__init__.py", line 14, in <module>
    from stable_baselines3.common.vec_env.vec_video_recorder import VecVideoRecorder
  File "/home/ubuntu/Downloads/deepbots-env/lib/python3.10/site-packages/stable_baselines3/common/vec_env/vec_video_recorder.py", line 4, in <module>
    from gymnasium.wrappers.monitoring import video_recorder
ModuleNotFoundError: No module named 'gymnasium.wrappers.monitoring'
```

any idea?

**Comment by wilhem** [comment]
Solved after downgrading gymnasium to 0.29.1

**Comment by tsampazk** [comment]
It's nice that you solved it and posted the solution, thank you. Sorry for the delayed answer. It seemed like a versioning issue. We will have to look into it anyway soon.
